src/renderer/services/wallet.js

```diff
@@ -127,7 +127,6 @@ export default class WalletService {
    * @returns {Boolean}
    */
   static canResignDelegate (wallet) {
-    console.log('canResignDelegate', wallet)
```

Comment:
Why we don't have a debug mode?

Reply:
That was debug code I added during development. Leaving stuff like that in would make the code messy. We use `logger.$error` for some areas where it might be useful (e.g. a try/catch) but that's all.

Reply:
It has already been removed!